src/BUILD.bazel

```diff
@@ -35,32 +36,36 @@ ts_devserver(
     # serve these files rooted at /
     additional_root_paths = [
         "npm/node_modules/zone.js/dist",
```

Comment:
Since we're injecting the `zone.min.js` and `deeppurple-amber.css` into the `index.html` file that step may also be able to add the additonal_root_paths for those scripts automatically. Then the only user specified additional_root_path would be `"npm/node_modules/@ngrx/store/bundles"` to go with `"@npm//node_modules/@ngrx/store:bundles/store.umd.min.js"` in data

Reply:
the problem with doing this automatically is we don't know what path prefix you want to strip.

you might want to serve it at `zone.js/dist/zone.js` in production for instance.
Or, we could totally remove these from additional_root_paths, making the BUILD file shorter but exposing `npm/node_modules/zone.js/dist/zone.js` as the served path.